lib/Mojo/Promise.pm

```diff
@@ -163,10 +165,12 @@ sub _all {
         sub {
           $results->[$i] = {status => 'fulfilled', value => [@_]};
           $all->resolve(@$results) if --$remaining <= 0;
+          ();
```

Comment:
No warnings were emitted before adding this line, but still thought it could be a good idea to include.